.github/workflows/test.yml

```diff
@@ -109,14 +112,20 @@ jobs:
       - name: Print CDK version
         run: poetry run cdk --version
 
+      - name: Get Oidc deploy role arn
+        run: cat .github/workflows/.env >> $GITHUB_ENV
+
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: ap-southeast-2
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI }}
+          role-to-assume: ${{ env.CiOidc }}
+
+      - name: Set up Boto3 with AssumeRoleWithWebIdentity for oidc connectivity
+        run: |
+          curl -H "Authorization: bearer $ACTIONS_ID_TOKEN_REQUEST_TOKEN" "$ACTIONS_ID_TOKEN_REQUEST_URL" \
+          | jq -r '.value' > /tmp/aws-eks-CiOidc-token
```

Comment:
This path isn't referenced anywhere in this PR - that seems like an oversight.

Reply:
Huh. You are right! Turns out I added this in https://github.com/linz/geostore/commit/54dd65527ace55b9d73b853b0fa6a0fd5ed38e88 in an attempt to use `AWS_WEB_IDENTITY_TOKEN_FILE` to allow s3 role assumption. This was fixed in [afbc25d](https://github.com/linz/geostore/pull/1822/commits/afbc25df6d00a2e9861c6fb80730a30258d18153), but the original addition was not removed.